diff --git a/EVERGI/forecaster_test.py b/EVERGI/forecaster_test.py
index f940cd4..4b8af7d 100644
--- a/EVERGI/forecaster_test.py
+++ b/EVERGI/forecaster_test.py
@@ -68,12 +68,12 @@ def build_model(l, drop, n, lr):
 
 if __name__ == '__main__':
     # FETCH THE DATASETS
-    dset = 'GEP'
-    country = 'Belgium'
+    dset = 'London'
+    country = 'UK'
     net = 'stlf'
-    LAYERS = 1
-    DROPOUT = 0
-    NEURONS = 64
+    LAYERS = 2
+    DROPOUT = 0.3
+    NEURONS = 128
     LR = 1e-3
     BATCHSIZE = 1500
 
@@ -95,6 +95,12 @@ if __name__ == '__main__':
             df = pd.read_csv(filename, index_col=0)
             datasets.append(df)
             names.append('B'+str(i))
+    if dset == 'London':
+        hourly = pd.read_csv('../data/London_smart_meters/London_hourly_all.csv', index_col='tstp')
+        for house in hourly['LCLid'].unique():
+            temp = hourly.loc[hourly['LCLid'] == house]
+            datasets.append(temp)
+            names.append(house)
         
     dX_train = []
     dT_train = []
@@ -126,6 +132,10 @@ if __name__ == '__main__':
         'patience': PATIENCE
     })
     config = wandb.config
+    if not os.path.exists('./results/'+dset+'/global'):
+        os.makedirs('./results/'+dset+'/global')
+    if not os.path.exists('./models/'+dset+'_models'):
+        os.makedirs('./models/'+dset+'_models')
 
     # full data LSTM MIMO compilation and fit
     LSTMIMO = build_model(l=LAYERS, drop=DROPOUT, n=NEURONS, lr=LR)
diff --git a/EVERGI/src/__pycache__/preprocessing_3days.cpython-38.pyc b/EVERGI/src/__pycache__/preprocessing_3days.cpython-38.pyc
index 263aa85..ff84ffd 100644
Binary files a/EVERGI/src/__pycache__/preprocessing_3days.cpython-38.pyc and b/EVERGI/src/__pycache__/preprocessing_3days.cpython-38.pyc differ
diff --git a/EVERGI/src/preprocessing_3days.py b/EVERGI/src/preprocessing_3days.py
index de41039..7ad9814 100644
--- a/EVERGI/src/preprocessing_3days.py
+++ b/EVERGI/src/preprocessing_3days.py
@@ -1,7 +1,7 @@
 import pandas as pd
 import numpy as np
 from workalendar.usa import Texas
-from workalendar.europe import Belgium
+from workalendar.europe import Belgium, UnitedKingdom
 from workalendar.usa import UnitedStates
 from workalendar.america import Canada
 
@@ -82,6 +82,8 @@ def preprocess(dataframe, country: str):
     # Setting the calendar holiday dates
     if country in ['Belgium', 'belgium']:
         cal = Belgium()
+    elif country in ['England', 'UK', 'uk', 'United Kingdom']:
+        cal = UnitedKingdom()      
     elif country in ['Texas', 'texas']:
         cal = Texas()
     elif country in ['United States', 'US', 'United States of America', 'us', 'usa', 'USA']:
diff --git a/EVERGI/wandb/debug-internal.log b/EVERGI/wandb/debug-internal.log
index e922cbe..df26451 120000
--- a/EVERGI/wandb/debug-internal.log
+++ b/EVERGI/wandb/debug-internal.log
@@ -1 +1 @@
-run-20210414_141653-2gow68ss/logs/debug-internal.log
\ No newline at end of file
+run-20210415_090600-1tzt7ibp/logs/debug-internal.log
\ No newline at end of file
diff --git a/EVERGI/wandb/debug.log b/EVERGI/wandb/debug.log
index bde7530..226d944 120000
--- a/EVERGI/wandb/debug.log
+++ b/EVERGI/wandb/debug.log
@@ -1 +1 @@
-run-20210414_141653-2gow68ss/logs/debug.log
\ No newline at end of file
+run-20210415_090600-1tzt7ibp/logs/debug.log
\ No newline at end of file
diff --git a/EVERGI/wandb/latest-run b/EVERGI/wandb/latest-run
index 0db4bb6..85ada83 120000
--- a/EVERGI/wandb/latest-run
+++ b/EVERGI/wandb/latest-run
@@ -1 +1 @@
-run-20210414_141653-2gow68ss
\ No newline at end of file
+run-20210415_090600-1tzt7ibp
\ No newline at end of file
